src/main/java/no/nav/tag/kontaktskjema/KontaktskjemaController.java

```diff
@@ -16,6 +16,7 @@
 @Slf4j
 public class KontaktskjemaController {
 
```

Comment:
shedlock-testen feilet da vi sjekket ut denne branchen. Det ser ut til at den feiler sporadisk. Burde sannsynligvis ta Thread.sleep(1100) eller noe for å være sikker på at basen har oppdatert seg.

Reply:
Ok!
Ja, den testen er jo skjør, og er helt avhengig av at jobben kjører som den skal. Vi bør endre det uansett, for vi kan ikke ha det sånn at vi blir bundet til å kjøre jobben hvert sekund bare for at testen skal funke / gå fort nok. Skal se om jeg finner en annen måte å teste det på.